index.html

```diff
@@ -0,0 +1,273 @@
+                                         Arcu pulvinar aenean nam laoreet nulla.
+                                    </p>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="col-2">
+                            <div class="testimonials__item">
+                                <div class="testimonials__photo">
+                                    <img src="./src/img/photo-2.png" alt="">
+                                </div>
+                                <div class="testimonials__info">
+                                    <span class="testimonials__name">Hanna Wilson</span>
+                                    <a class="testimonials__site" href="#" target="_blank" rel="noopener noreferrer">wordfaang.com</a>
+                                    <p class="testimonials__text">    
+                                        Nisl, sagittis viverra enim erat tortor ultricies massa turpis.
+                                         Arcu pulvinar aenean nam laoreet nulla.
+                                    </p>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </section>
+           </main>
+           <footer class="footer" id="sing-up">
+                <div class="footer__header row">
+                    <div class="col-2">
+                        <h2 class="footer__title">Try for free!</h2>
+                        <p class="footer__text">Get limited 1 week free try our features!</p>
+                    </div>
+                    <div class="col-2 footer__buttons">
+                        <a class="button button-pink" href="#">Learn more</a>
+                        <a class="button button-light" href="#">Request Demo</a>
+                    </div>
+                </div>
+                <div class="footer__navigation row">
+                    <div class="footer__contacts">
+                        <a class="logo footer__logo">
+                            <svg class="logo__icon" viewBox="0 0 45 36" fill="none">
+                                <rect y="15.5" width="18.3099" height="20" rx="9.15493" fill="#9C69E2"/>
+                                <rect x="26.5493" y="0.5" width="18.3099" height="35" rx="9.15493" fill="#F063B8"/>
+                            </svg>
+                            <span class="logo__text">DataWarehouse</span>
+                        </a>
+                        <p class="footer__addres">
+                            Warehouse Society, 234
+                            Bahagia Ave Street 
+                            PRBW 29281
+                        </p>
+                        <a class="footer__contact-link" href="mailto:info@warehouse.project">info@warehouse.project</a>
+                        <a class="footer__contact-link" href="tel:+1-232-3434">1-232-3434 (Main)</a>
```

Comment:
"-" should be replaced with "&#8209;" (non-breaking hyphen) in telephone numberhtml-validate(tel-non-breaking)